models/comments.model.js

```diff
@@ -0,0 +1,20 @@
+const db = require("../db/connection")
+
+const checkArticleExists = (id) => {
+    return db.query("SELECT * FROM articles WHERE article_id = $1",[id]).then(({ rows }) => {
```

Comment:
This should be extracted out into it's own util function as you will need this logic in a later ticket.